test/v1/validation/ConsolidatedPartnershipValidatorSpec.scala

```diff
@@ -46,13 +54,14 @@ class ConsolidatedPartnershipValidatorSpec extends BaseValidationSpec {
         leftSideError(model.validate).errorMessage shouldBe ConsolidatedPartnershipsNotSupplied(model).errorMessage
       }
 
-      "isElected is false and some partnerships are given" in {
-        val model = ConsolidatedPartnershipModel(isElected = false, isActive = true, consolidatedPartnerships = Some(Seq(PartnershipModel(partnershipName = CompanyNameModel("Partner 1"), sautr = Some(sautrFake)))))
-        leftSideError(model.validate).errorMessage shouldBe ConsolidatedPartnershipsSupplied(model).errorMessage
+      "isElected is false and no partnerships are given" in {
+        val model = ConsolidatedPartnershipModel(isElected = false, isActive = true, consolidatedPartnerships = None)
+        leftSideError(model.validate).errorMessage shouldBe ConsolidatedPartnershipsNotSupplied(model).errorMessage
       }
```

Comment:
I think we could do with more valid and invalid tests to cover all of the different possible permutations of isElected, isActive, partnerships.isDefined